src/Uno.UI/UI/Xaml/Shapes/Polyline.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Windows.UI.Xaml.Media;
+
+
+namespace Windows.UI.Xaml.Shapes
+{
+	public partial class Polyline : ArbitraryShapeBase
+	{
+		#region Points
+		public PointCollection Points
+		{
+			get { return (PointCollection)GetValue(PointsProperty); }
+			set { SetValue(PointsProperty, value); }
+		}
```

Comment:
We should find a way to cache the original string for wasm to prevent parsing and regenerating the same string again.